### DISCUSSION TITLE
Ability to publish script to multiple repo's

### DISCUSSION THREAD

**Discussion by sheldon-walters** [nicolaasburgers/definition-library-releases#17] [Ideas]
Consider adding the option to publish to multiple repo's -- whereby attempting to publish to a repo other than the original, would trigger the fork function.

**Comment by nicolaasburgers** [comment]
In the latest version (0.1.5) there is now the option to fork for an updated cluster that you already are the maintainer of:

![image](https://github.com/nicolaasburgers/definition-library-releases/assets/3597587/23d05e3c-24ad-456b-84a3-c4a06a069114)

Of course if you make a change to a cluster that you don't own, then it already by default offered a fork.  This now lets you create your own forks at will too, which is useful if you want to publish a variant to a public or other repo for collaboration with external parties, for example.
